src/DetailsView/components/settings-panel/settings/issue-filing/issue-filing-settings.tsx

```diff
@@ -2,42 +2,42 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { BugFilingSettingsContainer } from '../../../../../bug-filing/components/bug-filing-settings-container';
 import { FlaggedComponent } from '../../../../../common/components/flagged-component';
 import { FeatureFlags } from '../../../../../common/feature-flags';
 import { NamedSFC } from '../../../../../common/react/named-sfc';
+import { IssueFilingSettingsContainer } from '../../../../../issue-filing/components/issue-filing-settings-container';
 import { SettingsProps } from '../settings-props';
-import { GitHubBugSettingsUx } from './github-bug-settings-ux';
+import { GitHubIssueSettingsUx } from './github-issue-settings-ux';
```

Comment:
nit: should it be `GitHubIssueFillingSettingsUx`?

Reply:
same thing with the filename.

Reply:
I'm going to remove this file in the next PR, so